packages/block-library/src/navigation-link/index.php

```diff
@@ -154,9 +154,11 @@ function render_block_core_navigation_link( $attributes, $content, $block ) {
 
 	$wrapper_attributes = get_block_wrapper_attributes(
 		array(
-			'class' => $css_classes . ' wp-block-navigation-item' . ( $has_submenu ? ' has-child' : '' ) .
-				( $is_active ? ' current-menu-item' : '' ),
-			'style' => $style_attribute,
+			'class' => esc_attr(
```

Comment:
`get_block_wrapper_attributes` already does escaping, so the changes here are redundant and incorrect.

Reply:
> ...redundant and incorrect.

I appreciate this has come up a few times. I'm still going back through and removing these instances since we've now agreed several times that we don't need to escape inputs to this function.